### PR TITLE
Proper MySQL examples and syntax explanation

### PR DESCRIPTION
**Pull request by writeDavid**

### CODE REVIEW

File: plugins/sql/README.md [hunk 1]
```diff
@@ -65,6 +65,38 @@ const db = await Database.load("postgres://postgres:password@localhost/test");
...
+
+Since we use sqlx as our underlying library, we adopt their syntax. 
+- sqlite and postgres use the $# syntax when substituting query data
+- mysql uses ? when substituting query data
+
+```javascript
+[for sqlite & postgres]
+[INSERT example]
+    const result = await db.execute(
+        "INSERT into todos (id, title, status) VALUES ($1, $2, $3),
```
Comment (FabianLars) [inline plugins/sql/README.md]:
```suggestion
        "INSERT into todos (id, title, status) VALUES ($1, $2, $3)",
```

File: plugins/sql/README.md [hunk 2]
```diff
@@ -65,6 +65,38 @@ const db = await Database.load("postgres://postgres:password@localhost/test");
...
+[UPDATE example]
+    const result = await db.execute(
+        "UPDATE todos SET title = $1, completed = $2 WHERE id = $3",
+        [ todos.title, todos.status, todos.id ]
+    );
+
+[for mysql]
+[INSERT example]
+    const result = await db.execute(
+        "INSERT into todos (id, title, status) VALUES (?, ?, ?),
```
Comment (FabianLars) [inline plugins/sql/README.md]:
```suggestion
        "INSERT into todos (id, title, status) VALUES (?, ?, ?)",
```

**Review by FabianLars — CHANGES_REQUESTED** [review]
Thank you! Great contribution :)

Apart from the 2 missing `"`, could you also turn the titles into code comments (`//`) so that the syntax highlighting keeps working?

**Comment by writeDavid** [comment]
For the // conversion, do you want me to do that in both the index.ts and readme? 

MySQL is working flawlessly after figuring this out haha.  I was very confused at first.

**Comment by FabianLars** [comment]
Yes please. Just to be sure cause i don't know how all editors would render that code comment otherwise

**Review by FabianLars — DISMISSED** [review]

**Comment by FabianLars** [comment]
can you run `pnpm format` real quick too?

**Comment by writeDavid** [comment]
Okay. Ran that and pushed the updated file. 3rd times the charm??

**Comment by FabianLars** [comment]
wait, it checks the syntax in readme files but can't update them? 😕

**Comment by writeDavid** [comment]
I don't see any issues using pnpm format or format-check. I don't have a ton of prettier experience, so I don't know how to proceed.

**Review by FabianLars — DISMISSED** [review]
thanks again!

**Comment by FabianLars** [comment]
Or maybe not just yet 😂 The last commit wasn't signed so github doesn't allow me to merge the pr.

**Pull request closed without merging**